tests/drivers/dma/loop_transfer/src/dma.c

```diff
@@ -103,7 +103,7 @@ void main(void)
 		return;
 	}
 
-	k_sleep(SLEEPTIME);
+	k_sleep(K_MSEC(SLEEPTIME));
```

Comment:
could we have a` #define SLEEPTIME  K_MSEC(1000) `instead ?

Reply:
All works out, but I see most milliseconds time defines in the kernel use this format.